src/loaders/tvg/tvgTvgBinInterpreter.cpp

```diff
@@ -225,6 +225,12 @@ static unique_ptr<Fill> _parseShapeFill(const char *ptr, const char *end)
                 fillGrad->colorStops(stops, stopsCnt);
                 break;
             }
+#ifdef THORVG_LOG_ENABLED
+            default: {
+                TVGLOG("TVG", "Unsupported tag %d (0x%x) used as one of the fill properties, %d bytes skipped", block.type, block.type, block.length);
```

Comment:
but it's not an error. till now we didn't support TVG_TAG_FILL_TRANSFORM (#948) and now we do.
someone can open the tvg file produced with TVG_TAG_FILL_TRANSFORM flag with the old ThorVG version - this tag will be skipped, that's all. It would be nice to print a log about that, but is it an error?

Reply:
@mgrudzinska makes sense. thanks.